.github/workflows/package.yml

```diff
@@ -13,24 +13,42 @@ on:
       - release/**
     paths-ignore:
       - '.github/workflows/docusaurus.yml'
-      - 'docs/**'
-      - 'website/**'
   pull_request:
     types: [opened, reopened, edited, ready_for_review]
     branches:
       - main
-      - release/*
+      - release/**
     paths-ignore:
       - '.github/workflows/docusaurus.yml'
-      - 'docs/**'
-      - 'website/**'
-permissions:
```

Comment:
we still want default root permission to be read

Reply:
I'll get that back in

Reply:
done